.github/workflows/update-contributors.yaml

```diff
@@ -10,7 +10,7 @@ env:
 
 jobs:
   add-contributors:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
     steps:
```

Comment:
[nits]
The current description uses `ubuntu-latest`, so the latest version 24.04 is used, but I specified the version to be consistent with the other descriptions.